pkg/resources/openstack/lb.go

```diff
@@ -57,25 +62,28 @@ func (os *clusterDiscoveryOS) ListLB() ([]*resources.Resource, error) {
 			},
 		}
 		resourceTrackers = append(resourceTrackers, resourceTracker)
-	}
-	return resourceTrackers, nil
-}
 
-func (os *clusterDiscoveryOS) ListLBPools() ([]*resources.Resource, error) {
-	var resourceTrackers []*resources.Resource
+		//Identify pools associated to this LB
```

Comment:
octavia does not need any of these for loops (listeners, pools or monitors) so add if here something like execute only `if !os.osCloud.UseOctavia()`